TodoApp/Identifier/IdentifierType.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  IdentifierType.swift
+//  TodoApp
+//
+//  Created by koji torishima on 2020/09/21.
+//  Copyright © 2020 LeanjoyFC. All rights reserved.
+//
+
+import Foundation
+
+enum IdentifierType: String {
+    case segueId = "unwindByItemAdd"
+    case cellId = "Cell"
+    case nibId = "ItemListTableViewCell"
+}
```

Comment:
switchで条件網羅したい場面はないと思うので
```suggestion
enum IdentifierType {
    static let segueId = "unwindByItemAdd"
    static let cellId = "Cell"
    static let nibId = "ItemListTableViewCell"
}
```
でも良さそうに思いました。

Reply:
ありがとうございます！！
確かにそうですね！
詰めが甘かったです！
修正いたします！

Reply:
こちら修正いたしました。レビューありがとうございます！

Reply:
Identifierをこのようにまとめることができるのですね！
勉強になります！